src/org/labkey/test/tests/issues/IssueAPITest.java

```diff
@@ -419,28 +419,35 @@ public void testIssueWithoutAssignedTo() throws Exception
                 .setPriority(3L)
                 .setType("Defect");
 
-        var issueResponse= getIssueResponse( doIssueAction(issue));
-        assertEquals("should be assigned to guest", Long.valueOf(0), issueResponse.getAssignedTo());
-        assertEquals("should be open", "open", issueResponse.getStatus());
+        try
```

Comment:
This should have resulted in a validation error, since assigned to is a required field. Updating the test to reflect that.